proto/src/determined/api/v1/model.proto

```diff
@@ -218,8 +220,10 @@ message PostModelVersionResponse {
 message PatchModelVersionRequest {
   // The id of the model being updated.
   int32 model_id = 1;
+  // The id of the model version being updated.
+  int32 model_version_id = 2;
   // The model version being updated.
-  determined.model.v1.PatchModelVersion model_version = 2;
+  determined.model.v1.PatchModelVersion model_version = 3;
```

Comment:
you should keep the proto tag number the same as before when you "remove" a field or add new ones.
e.g. `model_version_id` here could be `= 3` or `= 30`, it doesn't have to be sorted in the code, but it has to be stable.

https://developers.google.com/protocol-buffers/docs/proto3#updating

it probably won't break anything for us because these messages are only used in json-serialized form, but I think we should stick to the guidelines anyway.